django-backend/fecfiler/contacts/views.py

```diff
@@ -58,6 +62,72 @@ class ContactViewSet(CommitteeOwnedViewSet):
     ]
     ordering = ["-created"]
 
+    @action(detail=False)
+    def candidate_lookup(self, request):
+        q = request.GET.get("q")
+        if q is None:
+            return HttpResponseBadRequest()
+
+        max_fec_results = self.get_int_param_value(
+            request, "max_fec_results", default_max_fec_results, max_allowed_results
+        )
+
+        max_fecfile_results = self.get_int_param_value(
```

Comment:
this code is repeated a handful of times in this file

Reply:
https://github.com/fecgov/fecfile-web-api/pull/357

Reply:
Resolved with patch